x-pack/plugins/observability_solution/logs_explorer/common/data_views/utils.ts

```diff
@@ -7,7 +7,7 @@
 
 export const buildIndexPatternRegExp = (basePatterns: string[]) => {
   // Create the base patterns union with strict boundaries
-  const basePatternGroup = `\\b(${basePatterns.join('|')})\\b[^,\\s]+`;
+  const basePatternGroup = `\\b(${basePatterns.join('|')})\\b([^,\\s]+)?`;
```

Comment:
**note**: update the regex also to match standalone indices with no wildcard patterns.